Capstone/Profile/AccountViewController.m

```diff
@@ -119,6 +119,12 @@ -(void)setUser {
     }
 }
 
+- (void)didMatchColleges:(BOOL)hide {
```

Comment:
the method name with parameter doesn't make a lot of sense at a glance; in this case, it'd make sense for the method name to be a verb, and the parameter to be the action (`didMatchColleges` - i forgot the grammatical term for what "did X" is).

i also wonder: is the parameter even necessary? i only see it being used in MatchVC [here](https://github.com/AbelKel/Capstone/blob/main/Capstone/Match%20View/MatchViewController.m#L163) — if we only want to call the delegate when matching has completed, then we can simplify the method

```suggestion
- (void)finishMatchingColleges {
```

Reply:
I'm just wondering that if I'm implementing the feedback below, I need to keep the method parameter as it was, right?